plugins/modules/sysctl.py

```diff
@@ -56,6 +56,16 @@
             - Verify token value with the sysctl command and set with -w if necessary
         type: bool
         default: 'no'
+    mode:
```

Comment:
why to not use the `add_file_common_args=True` and `module.load_file_common_arguments()` ? https://docs.ansible.com/ansible/latest/reference_appendices/module_utils.html

This would:
- avoid to duplicate documentation of same options
- keep consistent behavior between modules sharing such options
- extend the PR to other file attributes, ownership, selinux context...

Reply:
@quidame thank you for your valuable comments every time!
I'm currently working on fixing to use `add_file_common_args` and `load_file_common_arguments() based on your valuable advice.
For the options section in docstring, sysctl module doesn't handle all file attributes at this time, so I'm considering leaving the mode section as is.

Reply:
done!